tests/L0/run_transformer/run_layers_test.py

```diff
@@ -220,28 +220,36 @@ def test_column_parallel_linear(tensor_model_parallel_size):
     output_size_coeff = 17
     output_size = output_size_coeff * tensor_model_parallel_size
     batch_size = 7
+    hidden_size = 9
 
     # Network
-    identity_layer = IdentityLayer2D(batch_size, input_size).cuda()
+    identity_layer = IdentityLayer3D(batch_size, hidden_size, input_size).cuda()
     linear_layer = layers.ColumnParallelLinear(
```

Comment:
is `gradient_accumulation_fusion` set to `True` here?

Reply:
oh, I forgot to commit that change, appreciate it.